aws/offsite/aurora-snapshots-fargate/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM ubuntu:18.04
+FROM node:20
 
-RUN apt-get update && \
-    apt-get install -y \
-        nodejs \
-        npm
+WORKDIR /usr/src/app
 
-COPY . /root
-WORKDIR /root
+COPY package.json ./
 RUN npm install
-ENTRYPOINT node /root/restore-and-verify-backup.js
 
+COPY . .
```

Comment:
What is the purpose of this line?

Reply:
This replaced `COPY . /root`. It is copying everything from the context directory to the WORKDIR, which is now set prior to this line. This avoids hardcoding the path in multiple places. Similarly, I've switched the ENTRYPOINT to use a relative path instead of a full path.

Just for extra context, the other COPY line just brings over the package.json to support the `npm install`. This means a local developer can edit the source code all day long without invalidating the cache layers containing the npm install, making rebuilds much faster.

Reply:
to more succinctly answer your question - this line copies the source code into the container :)

Reply:
Hmm, I'm still a bit confused here; I might not be aware of the ways Docker's `COPY` differs from bash's `cp`. I would expect that `.` always refers to the current working directory, and so this would be copying everything from the current directory into the current directory. How does the first `.` and second `.` end up referring to different directories in this case?

Reply:
Ahhhh, that's the piece I was missing! That makes sense, then, thank you